types/index.d.ts

```diff
@@ -1156,6 +1156,11 @@ declare namespace RamdaAdjunct {
          * Identity type.
          */
         Identity: Function;
+
+        /**
+         * Converts input to an array.
+         */
+        toArray<T>(iterable: Iterable<T> | T): any[];
```

Comment:
@willcaul can you pls look into this signature and ts tests ? Could it be more specific ?